src/cmd/root.go

```diff
@@ -55,10 +60,10 @@ func init() {
 		// Re-add the original help function
 		originalHelp(c, s)
 	})
-
 	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
 	rootCmd.PersistentFlags().StringVarP(&zarfLogLevel, "log-level", "l", "", "Log level when running Zarf. Valid options are: warn, info, debug, trace")
 	rootCmd.PersistentFlags().StringVarP(&arch, "architecture", "a", "", "Architecture for OCI images")
+	rootCmd.PersistentFlags().StringVar(&zarfImageCache, "zarf-cache", config.ZarfDefaultImageCachePath, "Specify the location of the Zarf image cache")
```

Comment:
@jeff-mccoy I know you're trying to be mindful about how/when we add flags, especially if they're global flags. Do you think this flag is reasonable to be global? I think it should also be possible to only add the flag to the commands that use the .zarf-image-cache to avoid having a global flag at the cost of having more flags to maintain.

I also realized having a flag to dictate where the tmp directory gets created would also be needed to better solve the 'not enough space on root drive so I want to use a mounted drive' problem so I'm asking the above question twice I guess.

Reply:
@YrrepNoj this flag would only be used for `package create` operations so I think it should be under that probably.  You do lose the ability to do `zarf zarf.yaml` with this flag, but that's actually consistent with the rest of our behaviors already.

Reply:
Refactored so that the the new `zarf-cache` flag can only be used in `zarf package create` commands, also added an e2e test